src/types/resource.types.ts

```diff
@@ -16,17 +16,24 @@ export interface Resource {
   geographicalCoverages?: any[];
   observationalUnits?: any[];
   processMethods?: any[];
-  creators?: Creator[];
-  contributors?: Contributor[];
```

Comment:
Usikker men svarer ikke backend uten creators og contributors hvis de mangler? Per nå kan vi hvertfall ha det slik at vi ikke trenger å ha noen som helst contributors, og da lurer på om backend vil ikke sende contributor feltet overhodet.

Reply:
Forøvrig hvis creators og contributors blir satt til at de alltid skal være tilstede er det ganske mange plasser som har unødvendige sjekker om creator / contributor attributten eksisterer. Kanskje også fjerne de sjekkene?

Reply:
det blir nok ikke full sammenheng mellom datamodell i frontend og backend. Vi får passe på at de arrayene alltid er initiert om de mangler. Skal lete etter flere sjekker.

Reply:
Jeg lot en del av de arrayene som ikke er i bruk pr nå stå som mandatory i typescript - men de må også gjøres obligatoriske om når de evt skal brukes